### PR TITLE
[CWS] add troubleshooting section about security-agent flare

### PR DESCRIPTION
**Pull request by paulcacheux**
### Preview

https://docs-staging.datadoghq.com/paulcacheux/cws-secagent-flare
https://docs-staging.datadoghq.com/paulcacheux/cws-secagent-flare/agent/troubleshooting/send_a_flare/?tab=agentv6v7
https://docs-staging.datadoghq.com/paulcacheux/cws-secagent-flare/security_platform/cloud_workload_security/troubleshooting

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/security_platform/cloud_workload_security/troubleshooting.md [hunk 1]
```diff
@@ -4,11 +4,25 @@ kind: documentation
...
+
+| Platform     | Command                                                                             |
+| --------     | -------                                                                             |
+| Docker       | `docker exec -it datadog-agent security-agent flare <CASE_ID>`                      |
+| Kubernetes   | `kubectl exec -it <POD_NAME> -c security-agent -- security-agent flare <CASE_ID>`   |
+| Host         | `sudo /opt/datadog-agent/embedded/bin/security-agent flare <CASE_ID>`               |
+
+## Agent Self tests
+
+In order to ensure that the communication between the `security-agent` and the `system-probe` is working as expected and the Cloud Workload Security probe is able to detect system events, you can manually trigger self tests by running the following command:
```
Comment (n06) [inline content/en/security_platform/cloud_workload_security/troubleshooting.md:25]:
```suggestion
In order to ensure that the communication between the `security-agent` and the `system-probe` is working as expected and that Cloud Workload Security is able to detect system events, you can manually trigger self tests by running the following command:
```

**Review by n06 — CHANGES_REQUESTED** [review]
Looks good to me! Had one slight nit for you, but that's it

**Review by lebauce — APPROVED** [review]

**Review by n06 — APPROVED** [review]

File: content/en/security_platform/cloud_workload_security/troubleshooting.md [hunk 2]
```diff
@@ -4,11 +4,25 @@ kind: documentation
 description: "Troubleshooting for Cloud Workload Security."
 ---
 
-## Agent Self tests
+## Security Agent Flare
```
Comment (urseberry) [inline content/en/security_platform/cloud_workload_security/troubleshooting.md:7]:
```suggestion
## Security Agent flare
```

File: content/en/security_platform/cloud_workload_security/troubleshooting.md [hunk 3]
```diff
@@ -4,11 +4,25 @@ kind: documentation
...
+
+Similar to the [Agent flare][1], you can send necessary troubleshooting information to the Datadog support team with one flare command.
 
-In order to ensure that the communication between the `security-agent` and the `system-probe` is working as expected and the Cloud Workload Security probe is able to detect system events, you can manually trigger self tests by running the following command:
+If needed, the flare can be reviewed prior to sending since the flare prompts a confirmation before uploading it.
```
Comment (urseberry) [inline content/en/security_platform/cloud_workload_security/troubleshooting.md:11]:
```suggestion
The flare asks for confirmation before upload, so you may review the content before the Security Agent sends it.
```

**Review by urseberry — APPROVED** [review]

**Pull request merged**